operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1235,36 +1236,46 @@ spec:
                       value: "120000"
                     - name: STRIMZI_OPERATION_TIMEOUT_MS
                       value: "300000"
-                    - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
+                    - name: RELATED_IMAGE_STRIMZI_CLUSTER_OPERATOR_IMAGE
+                      value: registry.redhat.io/amq-streams/strimzi-rhel9-operator@sha256:06de6eaeb8cae4ea31f05a427bbe53f9d078de0030bd56fce4ecfbb5d0be461a
+                    - name: RELATED_IMAGE_STRIMZI_KAFKA_IMAGES_3_6_0
+                      value: registry.redhat.io/amq-streams/kafka-36-rhel9@sha256:177484ebf6f663eedfc558285e4e89e817277389250aec29323452082b6949e4
+                    - name: RELATED_IMAGE_STRIMZI_KAFKA_IMAGES_3_7_0
                       value: registry.redhat.io/amq-streams/kafka-37-rhel9@sha256:050aed7a91af406fbafcf6ce8fff4fa451cf4855f74a91aa081d794bf8c9aaa3
+                    - name: RELATED_IMAGE_STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
+                      value: registry.redhat.io/amq-streams/bridge-rhel9@sha256:4db7b231d68a244f5f71c8587123890e1344be933e1a030097c4b32a9bbfe32d
+                    - name: RELATED_IMAGE_STRIMZI_DEFAULT_MAVEN_BUILDER
+                      value: registry.redhat.io/amq-streams/maven-builder-rhel9@sha256:2393070a07677a482e90f53fef58f12e1c73963cf164738aaa1c095e5fe77d0d
+                    - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
+                      value: $(RELATED_IMAGE_KAFKA_3_7_0)
                     - name: STRIMZI_DEFAULT_KAFKA_EXPORTER_IMAGE
-                      value: registry.redhat.io/amq-streams/kafka-37-rhel9@sha256:050aed7a91af406fbafcf6ce8fff4fa451cf4855f74a91aa081d794bf8c9aaa3
+                      value: $(RELATED_IMAGE_KAFKA_3_7_0)
                     - name: STRIMZI_DEFAULT_CRUISE_CONTROL_IMAGE
-                      value: registry.redhat.io/amq-streams/kafka-37-rhel9@sha256:050aed7a91af406fbafcf6ce8fff4fa451cf4855f74a91aa081d794bf8c9aaa3
+                      value: $(RELATED_IMAGE_KAFKA_3_7_0)
```

Comment:
The related image used to be `strimzi-kafka-370` how does OSBS(?) know that  `KAFKA_3_7_0 -> strimzi-kafka-370`. As we have lost the `strimzi` prefix and added separators between the version components.

Reply:
I don't think the names of `relatedImages` is critical, but I could be wrong.

Reply:
> I don't think the names of relatedImages is critical, but I could be wrong.

We may need to update our STs that test mirroring to account for these name changes. Am I correct in my understanding Lukas? CC @im-konge